app/code/Magento/Sitemap/Model/Sitemap.php

```diff
@@ -362,12 +354,6 @@ protected function _initSitemapItems()
         ];
     }
 
-    /**
-     * Check sitemap file location and permissions
-     *
-     * @return \Magento\Framework\Model\AbstractModel
-     * @throws LocalizedException
-     */
```

Comment:
Can you explain why this doc block was removed?

Reply:
Replaced